panel/io/state.py

```diff
@@ -1093,7 +1097,12 @@ def user_info(self) -> Dict[str, Any] | None:
         """
         Returns the OAuth user information if enabled.
         """
+        is_guest = self.cookies.get('is_guest')
+        if is_guest:
+            return {"user": "guest"}
```

Comment:
I had also `username` as this is what the `authorize_callback` template needs. Otherwise it's None.